src/wordCount/Main.java

```diff
@@ -0,0 +1,55 @@
+package wordCount;
+
+import java.util.*;
+import java.lang.reflect.Array;
+
+public class Main {
+
+    public static int count(String[] words, String target) {
+        int count = 0;
+        for(int i = 0; i < words.length; i++) {
+            if(words[i].equals(target)) {
+                count++;
+            }
```

Comment:
Make sure to use `.equalsIgnoreCase` (instead of `.equals`, which treats `String` values with different casing as different) to ensure a more accurate word count.